Coding Style Guidline.rst

```diff
@@ -0,0 +1,223 @@
+pandas Coding Style Guide
+=========================
+
+1 Background
+------------
+
+Writing good code is not just about what you write. It is also about how
+you write it. During Continuous Integration testing, several tools will
+be run to check your code for stylistic errors. Generating any warnings
+will cause the test to fail. Thus, good style is a requirement for
+submitting code to *pandas*. This document serves as a supplement to the
+script demonstrating the proper coding style required for contributing
+to *pandas*.
+
+2 Patterns
+----------
+
+2.1 Header File

```

Comment:
What is this in reference to? This isn't anything Python related right? I think should just remove